src/applications/vaos/tests/list/video-appointments.unit.spec.jsx

```diff
@@ -425,3 +425,82 @@ describe('VAOS integration: upcoming video appointments', () => {
     expect(queryByText(/cancel appointment/i)).not.to.exist;
   });
 });
+
+describe('VAOS integration: upcoming ATLAS video appointments', () => {
+  it('should display ATLAS title', async () => {
+    const appointment = getVideoAppointmentMock();
+    const startDate = moment.utc().add(3, 'days');
+    appointment.attributes = {
+      ...appointment.attributes,
+      facilityId: '983',
+      clinicId: null,
+      startDate: startDate.format(),
+    };
+    appointment.attributes.vvsAppointments[0] = {
+      ...appointment.attributes.vvsAppointments[0],
+      dateTime: startDate.format(),
+      bookingNotes: 'Some random note',
+      appointmentKind: 'ADHOC',
+      status: { description: 'F', code: 'FUTURE' },
+
+      tasInfo: {
+        siteCode: '9931',
+        slotId: 'Slot8',
+        confirmationCode: '7VBBCA',
+        address: {
+          streetAddress: '114 Dewey Ave',
+          city: 'Eureka',
+          state: 'MT',
+          zipCode: '59917',
+          country: 'USA',
+          longitude: null,
+          latitude: null,
+          additionalDetails: '',
+        },
+        contacts: [
+          {
+            name: 'Decker Konya',
+            phone: '5557582786',
+            email: 'Decker.Konya@va.gov',
+          },
+        ],
+      },
+    };
+
+    mockAppointmentInfo({ va: [appointment] });
+
+    const screen = renderWithStoreAndRouter(<FutureAppointmentsList />, {
+      initialState,
+    });
+
+    await screen.findByText(/Video appointment at an ATLAS location/i);
+
+    // Should display appointment date
+    // expect(
+    //   screen.getByText((content, _element) =>
+    //     content.startsWith(startDate.format('dddd, MMMM D, YYYY ')),
+    //   ),
+    // ).to.be.ok;
```

Comment:
Can this be removed, or should we try to check for this date?

Reply:
Working on how to test this scenario when text node has embedded html elements.

Reply:
Fixed test case.